SpectralCubeReprojectMaskExample_v3.ipynb

```diff
@@ -0,0 +1,1546 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "cube1vel[:,125,125].to(u.K).with_spectral_unit(u.km/u.s).quicklook()\n",
+    "cube2vel_reproj[:,125,125].quicklook()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Do a quick continuum subtraction"
```

Comment:
This part should definitely be unpacked and explained over a few cells. I think it worth showing `med1` and `med2` so it's clear where the continuum sources are. IMO, it's enough to split into its own step.

Reply:
Does the median subtract lead to over subtraction for the wider line in orange?

Reply:
Not that I can tell. I think these cubes are already continuum-subtracted, because their spectra before and after the continuum subtraction look very similar. I added in a cell to look at the difference between the spectra before and after continuum subtraction so we can inspect this more.